### PR TITLE
Bugfix ktps 2298 feature mismatch

### PR DESCRIPTION
**Pull request by FrankKr**

### CODE REVIEW

File: openstef/pipeline/create_forecast.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def create_forecast_pipeline_core(
     data_with_features = OperationalPredictFeatureApplicator(
         # TODO use saved feature_names (should be saved while training the model)
         horizons=[pj["resolution_minutes"] / 60.0],
-        feature_names=model.feature_names,
+        feature_names=model.feature_importance_dataframe.index.values,
```
Comment (Davidswinkels) [inline openstef/pipeline/create_forecast.py:85]: Good fix! Is it possible to add a test (or multiple) to make sure we can avoid the bug in the future?

**Review by Davidswinkels — CHANGES_REQUESTED** [review]
See comment

File: openstef/pipeline/create_forecast.py (hunk 1 above)
Comment (FrankKr) [reply openstef/pipeline/create_forecast.py:85]: The tests were there, but the previously stored model was not compatible with those tests, as at was pretty old. 
I've updated the stored model to allow for accurate testing.

**Review by Davidswinkels — APPROVED** [review]
LGTM

**Pull request merged**